frontend/css/brainzplayer.less

```
@@ -351,6 +351,23 @@
   }
 }
 
+.volume {
+  position: fixed;
+  width: fit-content;
+  z-index: 109;
```

This will put the volume component on top of the queue if it is open, otherwise users might not see that there is a component at all and think the button does nothing.
```suggestion
  z-index: 110;
```